src/Twig/Extensions.php

```diff
@@ -254,15 +254,15 @@ public function money($amount, $currency = null)
 
         if ($this->locale !== $locale) {
             $this->locale = $locale;
-            $this->numberFormatter = new NumberFormatter($locale, NumberFormatter::DECIMAL);
+            $this->numberFormatter = new NumberFormatter($locale, NumberFormatter::CURRENCY);
         }
 
         $fractionDigits = Intl::getCurrencyBundle()->getFractionDigits($currency);
```

Comment:
I know you didn't write these lines ... but your PR made me curious and I had a look in `Symfony\Component\Intl\NumberFormatter\NumberFormatter` and it seems, as if there is already a `roundCurrency()` applied inside `formatCurrency()` so these lines could probably be deleted.
I need to do some testing with and without the original PHP intl extension.

Reply:
The reason why I left the rounding behavior is because the money function defaults to $currency=null.